jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -170,30 +165,13 @@ public void included()
     public void addCookie(HttpCookie cookie)
     {
         if (StringUtil.isBlank(cookie.getName()))
-        {
             throw new IllegalArgumentException("Cookie.name cannot be blank/null");
-        }
-        
-        if (getHttpChannel().getHttpConfiguration().getResponseCookieCompliance()==CookieCompliance.RFC2965)
-            addSetRFC2965Cookie(
-                cookie.getName(),
-                cookie.getValue(),
-                cookie.getDomain(),
-                cookie.getPath(),
-                cookie.getMaxAge(),
-                cookie.getComment(),
-                cookie.isSecure(),
-                cookie.isHttpOnly(),
-                cookie.getVersion());
-        else
-            addSetRFC6265Cookie(
-                cookie.getName(),
-                cookie.getValue(),
-                cookie.getDomain(),
-                cookie.getPath(),
-                cookie.getMaxAge(),
-                cookie.isSecure(),
-                cookie.isHttpOnly());
+
+        // add the set cookie
+        _fields.add(new SetCookieHttpField(cookie, getHttpChannel().getHttpConfiguration().getResponseCookieCompliance()));
+
+        // Expire responses with set-cookie headers so they do not get cached.
+        _fields.put(__EXPIRES_01JAN1970);
```

Comment:
Is this line new? Did the old code put in the default expiry of 01JAN1970?

Reply:
That line was present in 9.4.18.v20190429 too.

See: 
 * https://github.com/eclipse/jetty.project/blob/jetty-9.4.18.v20190429/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java#L342-L359
 * https://github.com/eclipse/jetty.project/blob/jetty-9.4.18.v20190429/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java#L422-L440